Program.cs

```diff
@@ -1,10 +1,181 @@
-﻿namespace Task1
+﻿using System;
+
+namespace Task1
 {
     internal class Program
     {
+        static void DisplayArray(int size, int[] array)
+        {
+            for (int i = 0; i < size; i++)
+            {
+                if (i < (size - 1))
+                {
+                    Console.Write($"{array[i]},");
+                }
+                else
+                {
+                    Console.Write($"{array[i]}]\n");
+                }
+            }
+        }
+
```

Comment:
Not good name for the method, you do not convert decimal to duodecimal here, you just check for AA in each number. Change the name

Reply:
Modified